package.json

```diff
@@ -8,16 +8,16 @@
   },
   "bugs": "https://github.com/TryGhost/Ghost/issues",
   "private": true,
-  "version": "0.11.3",
+  "version": "0.11.8",
   "dependencies": {
-    "casper": "TryGhost/Casper#1.3.4",
-    "ghost": "0.11.3",
+    "casper": "TryGhost/Casper#1.3.6",
+    "ghost": "0.11.8",
     "ghost-s3-storage-adapter": "3.0.4",
     "ncp": "^2.0.0",
     "pg": "latest"
   },
   "engines": {
-    "node": "~0.12.0"
+    "node": "^6.10.3"
```

Comment:
What are your thoughts on simply using the version string straight from [Ghost’s package.json](https://github.com/TryGhost/Ghost/blob/0.11.8/package.json#L26) (i.e. going with `^4.2.0 || ^6.9.0`)?